components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts

```diff
@@ -22,49 +25,111 @@ export class AutomateSettingsRequests {
+      infra : {
+        job_settings: [
+          {
+            name: 'periodic_purge_timeseries',
+            purge_policies: {
+              elasticsearch: []
+            }
+          }
+        ]
+      },
+      compliance: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
+      },
+      event_feed: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
       }
-      case IngestJobs.DeleteNodes: {
-        url = `${RETENTION_URL}/nodes/delete-nodes/config`;
-        break;
+    };
+
+    jobs.forEach((job: IngestJob): void => {
+      let thisJob;
+      let matchedNest;
+
+      switch (job.name) {  // ADD STRONG TYPING
+        case 'delete_nodes':      // fallthrough
+        case 'missing_nodes':     // fallthrough
+        case 'missing_nodes_for_deletion':
+          thisJob = this.unfurlIngestJob(job);
+          body[job.category].job_settings.push(thisJob);
+          break;
+
+        case 'periodic_purge_timeseries': // fallthrough
+        case 'periodic_purge':
+          thisJob = this.unfurlIngestJob(job, true);
+          matchedNest = body[job.category].job_settings.find(item => item.name === job.name);
+          matchedNest.purge_policies.elasticsearch.push(thisJob);
+          break;
+
+        default:
+          break;
       }
-      default:
-        return;
-    }
 
-    const body = {
-      'threshold': job.threshold,
-      'running': job.running
-      // We can also modify how often the job runs (every X time)
-      // but we don't need that now!
-      // 'every': job.every
-    };
+    });
 
-    return this.http.post<any>(url, body);
+    return this.http.put<any>(url, body);
   }
 
+  private unfurlIngestJob(job: IngestJob, nested: boolean = false): UnfurledJob {
+    if (nested) {
+      return {
+        policy_name: job.nested_name,
+        older_than_days: parseInt(job.threshold, 10),
+        disabled: job.disabled
+      };
+    } else {
+      return {
+        name: job.name,
+        threshold: job.threshold,
+        disabled: job.disabled
+      };
+    }
+  }
+
+
   private convertResponseToJobSchedulerStatus(
     respJobSchedulerStatus: RespJobSchedulerStatus): JobSchedulerStatus {
-      const jobs = respJobSchedulerStatus.jobs.map((respJob: RespJob) => new IngestJob(respJob));
-      return new JobSchedulerStatus(respJobSchedulerStatus.running, jobs);
+
+    const allJobs = [];
+
+    Object.keys(respJobSchedulerStatus).forEach((category: string) => {
+      if (respJobSchedulerStatus[category]) {
+        const catJobs = respJobSchedulerStatus[category].jobs
+          .map((respJob: RespJob) => new IngestJob(category, respJob));
+        allJobs.push(...catJobs);
+      }
+    });
+
+    return new JobSchedulerStatus(allJobs);
   }
```

Comment:
This functions purpose is to take the response we get back when requesting the configurations, and untangling it so that we can place it back in the form for the user to see.  I had to start by separating them by category, `infra`, `compliance`, `services`, `event_feed`.